### ISSUE TITLE
Remove explicit Promise return in Session process function

### ISSUE THREAD

**Issue by poftadeh** [Seneca-CDOT/plumadriver#50]
**What would you like to be changed**:

In `Session.ts`:

```js
async process({
    command,
    parameters,
    urlVariables,
  }: Pluma.Request): Promise<string> {
    let response = null;

    return new Promise(
      async (resolve, reject): Promise<void> => ...
```
Async functions wrap return values in Promises, so this can be omitted.

**Comment by c3ho** [comment]
Hey, I worked on telescope to do this sort of stuff. I can take this issue on.

**Comment by poftadeh** [comment]
Resolved by #143
